api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/context/MessageContextLoader.java

```diff
@@ -112,6 +113,23 @@ public EmailMessageContext loadEmailMessageContext(Message message) {
                 .build();
     }
 
+    /**
```

Comment:
Yes, I think a single class level comment will replace the need for any method level JavaDoc.